spec/features/items/index_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe 'Items Index Page', type: :feature do
+  describe 'As a visitor' do
+    before(:each) do
+      @supermarket_1 = Supermarket.create!(name: "Ralphs", location: "Over There St.")
+      @supermarket_2 = Supermarket.create!(name: "Dougs", location: "Over Here St.")
+      @customer_1 = Customer.create!(name: "Alex Long")
+      @customer_2 = Customer.create!(name: "Alex Short")
+      @customer_3 = Customer.create!(name: "Alex Medium")
+      @item_1 = @supermarket_1.items.create!(name: "Apple", price: 5)
+      @item_2 = @supermarket_1.items.create!(name: "Banana", price: 2)
+      @item_3 = @supermarket_1.items.create!(name: "Orange", price: 3)
+      @item_4 = @supermarket_2.items.create!(name: "Onion", price: 1)
+      @item_5 = @supermarket_2.items.create!(name: "Bell Pepper", price: 4)
+      @customer_item_1 = CustomerItem.create!(customer_id: @customer_1.id, item_id: @item_1.id)
+      @customer_item_2 = CustomerItem.create!(customer_id: @customer_1.id, item_id: @item_2.id)
+      @customer_item_3 = CustomerItem.create!(customer_id: @customer_2.id, item_id: @item_3.id)
+      @customer_item_4 = CustomerItem.create!(customer_id: @customer_1.id, item_id: @item_3.id)
+      @customer_item_5 = CustomerItem.create!(customer_id: @customer_3.id, item_id: @item_3.id)
+      @customer_item_6 = CustomerItem.create!(customer_id: @customer_3.id, item_id: @item_4.id)
+
+      visit "/items"
+    end
+
+    it 'shows all items' do
+      expect(page).to have_content(@item_1.name)
+      expect(page).to have_content(@item_2.name)
+      expect(page).to have_content(@item_3.name)
+      expect(page).to have_content(@item_4.name)
+      expect(page).to have_content(@item_5.name)
+    end
+
+    it 'shows all items prices, number of customers that have purchased them, and the supermarket they belong to' do
+      within "#item-#{@item_1.id}" do
+        expect(page).to have_content(@item_1.price)
+        expect(page).to have_content(@supermarket_1.name)
+        expect(page).to have_content("Purchased by 1 customer(s)")
+      end
+      within "#item-#{@item_2.id}" do
+        expect(page).to have_content(@item_2.price)
+        expect(page).to have_content(@supermarket_1.name)
+        expect(page).to have_content("Purchased by 1 customer(s)")
+      end
+      within "#item-#{@item_3.id}" do
+        expect(page).to have_content(@item_3.price)
+        expect(page).to have_content(@supermarket_1.name)
+        expect(page).to have_content("Purchased by 3 customer(s)")
+      end
+      within "#item-#{@item_4.id}" do
+        expect(page).to have_content(@item_4.price)
+        expect(page).to have_content(@supermarket_2.name)
+        expect(page).to have_content("Purchased by 1 customer(s)")
+      end
+      within "#item-#{@item_5.id}" do
+        expect(page).to have_content(@item_5.price)
+        expect(page).to have_content(@supermarket_2.name)
+        expect(page).to have_content("Purchased by 0 customer(s)")
+      end
+    end
```

Comment:
Just reiterating my ask from earlier to use "anchor" text. Other than this, this test suite is great!